src/lib.rs

```diff
@@ -1144,21 +1150,15 @@ pub mod pallet {
 				dispute_id,
 				query_id,
 				timestamp,
-				reporter: dispute_initiator.clone(),
+				reporter: dispute_initiator,
 			});
 
-			// Lookup corresponding addresses on controller chain
```

Comment:
Restore comment as '// Lookup corresponding address on controller chain' please.

Reply:
merged both contexts(this comment and previous one) on the same comment.